src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -51,53 +51,9 @@ void ForceRunCancelImpl(ForceRunCancelCommand command)
 
         void CreateNodeImpl(CreateNodeCommand command)
         {
-            var node = command.Node;
+            var node = GetNodeFromCommand(command);
```

Comment:
move out all these if-else statements into a separate method to avoid many indents because of them